### PR TITLE
Hope it’s ok

### PR DESCRIPTION
**Pull request by acaliaro**
If you are submitting a new blog please read:

By sending this pull request to add my blog I verify that I adhere to the Planet Xamarin blog guidelines:

- I have a valid blog & RSS URL
- Host NO malicious or offensive content on the blog (including photos, swearing, etc.)
- Blog is active with at least 3 Xamarin related blog posts in the last 6 months
- If the blog has mixed content (Xamarin and Personal/Non-Xamarin blogs) a filter has been applied
- If you delete your blog you will come delete your blog from Planet Xamarin
- Your blog may be removed at any time if any of these are broken.

### CODE REVIEW

File: src/Firehose.Web/Web.config [hunk 1]
```diff
@@ -37,10 +36,6 @@
   <system.web>
     <compilation debug="true" targetFramework="4.5.1">
       <assemblies>
-        <add assembly="System.Globalization, Version=4.0.0.0,
```
Comment (Cheesebaron) [inline src/Firehose.Web/Web.config]: Please dont change these

File: src/Firehose.Web/Authors/AlessandroCaliaro.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        public string FirstName => "Alessandro";
+        public string LastName => "Caliaro";
+        public string ShortBioOrTagLine => "software developer for mobile devices";
+        public string EmailAddress => "acaliaro@libero.it";
+        public string TwitterHandle => "acaliaro";
+        public string GravatarHash => "a7466eb1c467806f77bc692a4745d0f9";
+        public string StateOrRegion => "Lissone, Italy";
+        public Uri WebSite => new Uri("http://acaliaro.wordpress.com");
+        public string GitHubHandle => "acaliaro";
+		public GeoPosition Position => new GeoPosition(45.6327830, 9.2272650);
```
Comment (Cheesebaron) [inline src/Firehose.Web/Authors/AlessandroCaliaro.cs]: Derped indentation

File: src/Firehose.Web/Authors/AlessandroCaliaro.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+using System.ServiceModel.Syndication;
+using Firehose.Web.Infrastructure;
+
+namespace Firehose.Web.Authors
+{
+    public class AlessandroCaliaro : IAmACommunityMember
+    {
+        public string FirstName => "Alessandro";
+        public string LastName => "Caliaro";
+        public string ShortBioOrTagLine => "software developer for mobile devices";
```
Comment (Cheesebaron) [inline src/Firehose.Web/Authors/AlessandroCaliaro.cs]: Maybe write something more exciting. Also unclear what you mean here. This implies that you develop mobile devices.

Comment (acaliaro) [inline src/Firehose.Web/Authors/AlessandroCaliaro.cs]: @Cheesebaron English is my big problem ;)

File: src/Firehose.Web/Web.config (hunk 1 above)
Comment (acaliaro) [inline src/Firehose.Web/Web.config]: I have not touch this. I have only add my AlessandroCaliaro.cs.
What Should I do?

File: src/Firehose.Web/Authors/AlessandroCaliaro.cs (hunk 2 above)
Comment (acaliaro) [inline src/Firehose.Web/Authors/AlessandroCaliaro.cs]: I don't see this bad indentation in my source code...

Comment (acaliaro) [inline src/Firehose.Web/Authors/AlessandroCaliaro.cs]: ok, you use spaces.

**Comment by acaliaro** [comment]
Can this be merged?

**Pull request merged**